ui/src/message_popup/renderers/minimal_timed_view.jsx

```diff
@@ -0,0 +1,88 @@
+/* @flow weak */
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+var sceneTimer;
+
+// This response supports responding with text.
+export default React.createClass({
+  displayName: 'MinimalTimedView',
+
+  propTypes: {
+    onLogMessage: React.PropTypes.func.isRequired,
+    onResponseSubmitted: React.PropTypes.func.isRequired,
+    recordText: React.PropTypes.string,
+  },
+
+  getDefaultProps() {
+    return {
+      recordText: 'Respond',
+      autoFocus: true,
+      textHeight: 48
+    };
+  },
+
+
+  getInitialState() {
+    sceneTimer = setTimeout(this.onTimer, 25000);
+    return {
+      responseText: 'type here!'
+    };
+  },
+
+  onTextChanged(e) {
+    const responseText = e.target.value;
+    this.setState({responseText});
+  },
+
+  onTimer() {
+    const {responseText} = this.state;
+    this.props.onLogMessage('message_popup_text_response', {responseText});
```

Comment:
This is recording something slightly different than a text response.  Let's change the `message_popup_text_response` message type to match whatever the component name is (eg., `timer_auto_advance_response`.  Also, `responseText` is meaningless here (the user isn't typing anything in this component) so we can remove those bits too.